_docs_agent/host-feature.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: HOST feature
+description: Monitor services that run on a host from our Docker plugin
```

Comment:
I like the description that was using in index.md:

Monitor services that run on a host from our Container Agent.